packages/client/src/rpc/validation.ts

```diff
@@ -1,18 +1,21 @@
 import { INVALID_PARAMS } from './error-code.js'
 
+type RPCMethod = (params: any[]) => any
```

Comment:
```suggestion
export type RPCMethod = (...params: any) => any
```

Let's move this to `./types.ts` in the same directory and then use it to type the `callWithStackTrace` method for consistency.  Should have the same effect overall.

Reply:
@acolytec3 addressed